cpp/ql/src/semmle/code/cpp/MemberFunction.qll

```diff
@@ -1,4 +1,9 @@
-import cpp
+/**
+ * Provides classes for working with C++ constructors, destructors,
+ * and user-defined operators.
+ */
```

Comment:
Shouldn't this also mention member functions somewhere? Like:

```codeql
/**
 * Provides classes for working with C++ member functions, constructors, destructors,
 * and user-defined operators.
 */
```

Reply:
Yes it clearly should - thank you.  Done.